packages/gatsby/src/commands/develop.ts

```diff
@@ -124,6 +125,7 @@ module.exports = async (program: IProgram): Promise<void> => {
     programPath: program.directory,
   })
 
+  await startGraphQLServer(program.directory, true)
```

Comment:
I don't think develop should start recipes server, rather it should be the wrapper. You don't need to wait for develop to run. It will also allow us to not depend on `gatsby-recipes`.

Reply:
This is the wrapper :blush:

Reply:
Yeah, got it. Now I think that it should be outside of Gatsby.